turtlebot3_sim_tests/resources/turtlebot_nav.rossystem

```diff
@@ -0,0 +1,617 @@
+RosSystem { Name 'demo'
+    RosComponents ( 
+
+        ComponentInterface { name '/gazebo'
+            RosParameters {
+                RosParameter '/gazebo/sor_pgs_precon_iters' {RefParameter './gazebo./gazebo./gazebo/sor_pgs_precon_iters' value 0},
+                RosParameter '/gazebo/sor_pgs_w' {RefParameter './gazebo./gazebo./gazebo/sor_pgs_w' value 1.4},
+                RosParameter '/gazebo/auto_disable_bodies' {RefParameter './gazebo./gazebo./gazebo/auto_disable_bodies' value False},
+                RosParameter '/gazebo/sor_pgs_rms_error_tol' {RefParameter './gazebo./gazebo./gazebo/sor_pgs_rms_error_tol' value 0.0},
+                RosParameter '/gazebo/gravity_z' {RefParameter './gazebo./gazebo./gazebo/gravity_z' value -9.8},
+                RosParameter '/gazebo/contact_surface_layer' {RefParameter './gazebo./gazebo./gazebo/contact_surface_layer' value 0.01},
+                RosParameter '/gazebo/max_update_rate' {RefParameter './gazebo./gazebo./gazebo/max_update_rate' value 1000.0},
+                RosParameter '/gazebo/cfm' {RefParameter './gazebo./gazebo./gazebo/cfm' value 1e-05},
+                RosParameter '/gazebo/erp' {RefParameter './gazebo./gazebo./gazebo/erp' value 0.2},
+                RosParameter '/gazebo/gravity_x' {RefParameter './gazebo./gazebo./gazebo/gravity_x' value 0.0},
+                RosParameter '/gazebo/time_step' {RefParameter './gazebo./gazebo./gazebo/time_step' value 0.001},
+                RosParameter '/gazebo/gravity_y' {RefParameter './gazebo./gazebo./gazebo/gravity_y' value 0.0},
+                RosParameter '/gazebo/enable_ros_network' {RefParameter './gazebo./gazebo./gazebo/enable_ros_network' value True},
+                RosParameter '/gazebo/contact_max_correcting_vel' {RefParameter './gazebo./gazebo./gazebo/contact_max_correcting_vel' value 2000.0},
+                RosParameter '/gazebo/max_contacts' {RefParameter './gazebo./gazebo./gazebo/max_contacts' value 20},
+                RosParameter '/gazebo/sor_pgs_iters' {RefParameter './gazebo./gazebo./gazebo/sor_pgs_iters' value 150}}
+            RosPublishers {
+                RosPublisher '/odom' {RefPublisher './gazebo./gazebo./odom'},
```

Comment:
Similar to ROSParameter, RosPublisher also has RefPublisher './gazebo./gazebo./odom'. Is it should be 'gazebo.gazebo.odom'?